config/initializers/constants.rb

```diff
@@ -1,3 +1,4 @@
+TITLE_OPTIONS = %w(Mr Mrs Miss Ms)
```

Comment:
How will we do translations?

Reply:
No issues.. We are not handling translation here.